<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>655</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MIDI Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Changes</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="browseLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Select a .mid File</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="functionLayout">
     <item>
      <widget class="QLabel" name="FunctionLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Select Function</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="functionSelectBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Strummify</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Change Delta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Change Note</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Change Volume</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>541</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="dynFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="trackLabel">
       <property name="text">
        <string>Track</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="trackSelectBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="measuresBox">
       <property name="text">
        <string>Measures</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="startSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="stopSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Time Sig</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="noteCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Note</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="noteComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="sigABox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>/</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sigBBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>4</number>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Output</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>541</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Randomize Value?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>When selected, the amount slider will apply a random value change within range set by slider values. Otherwise, applied value will be constant.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="magnitudeLabel">
       <property name="text">
        <string>Magnitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="magnitudeSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lowRangeLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Low Magnitude</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="lowRangeSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="slideLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Slide</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="slideComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>From 0 to magnitude</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>From current to magnitude</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Values</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Where to apply values</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overwriteButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep Changes</string>
    </property>
   </widget>
   <widget class="QLabel" name="saveLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMIDI_Editor">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMIDI_Editor"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
